content/departments/product-engineering/engineering/cloud/delivery/index.md

```diff
@@ -16,32 +16,56 @@ Enable any Sourcegraph customer or user to trial or run (in production) Sourcegr
 
 ## Responsibilities
 
-- Deployments methods
+- Deployments methods (e.g. docker, kustomize, helm)
 - [Customer deployments and docs](https://docs.sourcegraph.com/admin/install)
 - [Managed instances](./managed/index.md)
-- ~~[Sourcegraph releases](../../process/releases/index.md)~~
+- ~~[Sourcegraph releases](../../process/releases/index.md)~~ releases are owned by the [release guild](../../process/releases/release_guild.md)
 - Deployment configuration and configuration orchestration ([Kustomize](https://docs.sourcegraph.com/admin/install/kubernetes))
 
-Specifically not included:
+## Contact
 
-- The release pipeline that serves sourcegraph.com - [Cloud Software-as-a-Service Team](../../cloud/devops/index.md)
```

Comment:
I don't think it's worth mentioning this anymore since Delivery and DevOps are two teams now